plasma_framework/migrations/1040_setup.js

```diff
@@ -17,7 +17,7 @@ module.exports = async (
     [deployerAddress, maintainerAddress, authorityAddress],
 ) => {
     const vault = process.env.VAULT || false;
-    if (!vault) {
+    if (vault === false) {
```

Comment:
wait shouldn't this be if (vault !== true)

Reply:
That doesn't work either, because if you set `VAULT=true` in the env, javascript `vault !== true` resolves to true.
Which is kind of unexpected...

Reply:
omg yes 🤦